SensusService/Probes/User/Scripts/ScriptRunner.cs

```diff
@@ -415,52 +446,37 @@ private void StartRandomTriggerCallbacks()
 #error "Unrecognized platform."
 #endif
 
-                // find next future trigger window, ignoring month, day, and year of windows. the windows are already sorted.
+                // find number of days left in participation horizon
```

Comment:
Alternatively we could check callbacks and reschedule if needed at some interval, in the same way the script tray checks for aged out scripts every second?